### ISSUE TITLE
Missing py.typed

### ISSUE THREAD

**Issue by Goldziher** [cunla/fakeredis-py#63]
Hiya and thanks for this library. 

The issue is that it doesn't play nice with type checkers - because its missing a `py.typed` file. The fx is simple - add a `py.typed` file (its an empty file) into the source folder of the lib, and bundle it as part of the wheel. You can find instructions in the mypy docs.

Cheers.

**Comment by cunla** [comment]
Can you provide what is the issue exactly? what type-checker are you running? how are you running it? etc.?

**Comment by Goldziher** [comment]
I'm running mypy. This is a standard mypy requirement - https://mypy.readthedocs.io/en/stable/installed_packages.html

**Comment by cunla** [comment]
I am not experienced with mypy, how do you run it? I need instructions how to replicate..

**Comment by the-wondersmith** [comment]
@Goldziher I've discussed with with @cunla and the resolution we came to was that we'd get type annotations added to `fakeredis` when and as time allows. @cunla did not think it would be feasible to type-annotate the entire `fakeredis` codebase in a single pass, and (though I tried) I couldn't poke any holes in his logic.

tl;dr typing will be added eventually, and is fairly high on _my_ wishlist, but it won't be an instant thing.
